README.md

```diff
@@ -121,6 +121,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Open Library](https://openlibrary.org/developers/api) | Books, book covers and related data | No | Yes | No |
 | [Penguin Publishing](http://www.penguinrandomhouse.biz/webservices/rest/) | Books, book covers and related data | No | Yes | Unknown |
 | [Rig Veda](https://aninditabasu.github.io/indica/html/rv.html) | Gods and poets, their categories, and the verse meters, with the mandal and sukta number | No | Yes | Unknown |
+| [Shrimad Bhagavad Gita](https://vedicscripturesapi.herokuapp.com/) | Open Source Shrimad Bhagavad Gita API including 21+ authors translation in Sanskrit/English/Hindi | No | Yes | Yes |
```

Comment:
The url of `https://vedicscripturesapi.herokuapp.com/` to `https://vedicscriptures.github.io/`. 
Can you please replace the current link with `https://vedicscriptures.github.io/`?

Reply:
Updated as per request

Check on: https://github.com/PtPrashantTripathi/public-apis/pull/1#issue-616159400